x/evidence/keeper/infraction.go

```diff
@@ -26,19 +27,6 @@ func (k Keeper) HandleEquivocationEvidence(ctx sdk.Context, evidence *types.Equi
 	logger := k.Logger(ctx)
 	consAddr := evidence.GetConsensusAddress()
 
-	if _, err := k.slashingKeeper.GetPubkey(ctx, consAddr.Bytes()); err != nil {
```

Comment:
I might have asked this before, but is it okay that we remove this from the evidence module on the provider chain?